simpler_eda/numerical_eda.py

```diff
@@ -1,14 +1,21 @@
 import altair as alt
 import numpy as np
 import pandas as pd
+from vega_datasets import data
+import warnings
+import pytest
+from pandas.api.types import is_numeric_dtype
 
 
-def numerical_plot(
+cars = data.cars()
+
+
+def numerical_eda(
     data,
     xval=None,
     yval=None,
     plot_type="scatter",
-    color=None,
+    color="red",
```

Comment:
For color here ( color="red"), we should give a column name in str or None here?